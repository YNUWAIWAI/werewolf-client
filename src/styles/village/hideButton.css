.vi--hide-button {
  grid-area: command;
  position: relative;
  z-index: var(--hide-button--z-index);

  &--button {
    background: rgb(48, 48, 48);
    border: 0;
    border-radius: 4px;
    height: var(--hide-button-height);
    left: 0;
    padding: 0;
    position: absolute;
    top: -24px;
    width: 48px;

    &:focus {
      outline: unset;
    }

    &.hide {
      bottom: 0;
      top: unset;

      & > svg {
        transform: rotate(0.5turn);
      }
    }

    & > svg {
      fill: white;
      width: 100%;
    }
  }
}
