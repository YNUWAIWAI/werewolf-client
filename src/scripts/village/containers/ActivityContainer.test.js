import ActivityContainer from './ActivityContainer'
import React from 'react'
import {initialState} from '../reducers/hideButton'
import {shallow} from 'enzyme'

test('<ActivityContainer /> initialState', () => {
  const getState = () => ({
    hideButton: initialState
  })
  const subscribe = jest.fn()
  const store = {
    getState,
    subscribe
  }
  const wrapper = shallow(<ActivityContainer store={store} />)

  expect(wrapper.props().expand).toBe(false)
})
