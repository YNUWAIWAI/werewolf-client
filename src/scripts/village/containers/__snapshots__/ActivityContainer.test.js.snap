// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActivityContainer /> expand: false 1`] = `"<div class=\\"activity \\"><div class=\\"chat\\"></div></div>"`;

exports[`<ActivityContainer /> expand: true 1`] = `"<div class=\\"activity expand\\"><div class=\\"chat\\"></div></div>"`;
